### PR TITLE
Admin privilege in keycloak to user for nexus doc added

### PR DESCRIPTION
**Pull request by TehreemNisa**

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-266-12b739e1`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-266-129db777`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-266-6388fb1a`

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]
1. Crop the URL/bookmarks bar of keycloak-1.png since it has the nonprod URL.
2. You need to add a reference of this page in .vuepress/config.js
3. Also provide a link to jump on this page from this page apropriately: https://docs.cloud.stakater.com/content/sre/repository/03-permissions.html

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-266-f45d3032`

File: .vuepress/config.js [hunk 1]
```diff
@@ -103,7 +103,8 @@ module.exports = {
           "/content/sre/repository/06-accessing-repository.md",
           "/content/sre/repository/03-permissions.md",
           "/content/sre/repository/04-routes.md",
-          "/content/sre/repository/05-FAQ.md"
+          "/content/sre/repository/05-FAQ.md",
+          "/content/sre/repository/08-grant-nexus-admin-keycloak.md"
```
Comment (usamaahmadkhan) [inline .vuepress/config.js]: This should come affter `03-permissions.md`

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-266-4b1c87b7`

File: content/sre/repository/03-permissions.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ For human users which login via SSO we have following roles available.
 | nexus-oauth-viewer  |nx-repository-view-*-*-browse, nx-repository-view-*-*-read |
 | nexus-oauth-editor  |nx-datastores-all, nx-blobstores-all, nx-analytics-all, nx-repository-admin-*-*-* |
 
-On first login you automatically get `nexus-oauth-viewer`
+On first login you automatically get `nexus-oauth-viewer`. See [Admin privilege to user for nexus on Keycloack](../repository/08-grant-nexus-admin-keycloak.md) on how to configure admin role with keycloak for nexus.
```
Comment (usamaahmadkhan) [inline content/sre/repository/03-permissions.md]: See [ Granting Admin privilege to user for nexus on Keycloak] (../repository/08-grant-nexus-admin-keycloak.md) on how to configure admin role with keycloak for nexus.

File: content/sre/repository/08-grant-nexus-admin-keycloak.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+## Admin privilege to user for nexus on Keycloack
```
Comment (usamaahmadkhan) [inline content/sre/repository/08-grant-nexus-admin-keycloak.md]: Granting Admin privilege to user for nexus on Keycloak

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: content/sre/repository/03-permissions.md (hunk 2 above)
Comment (TehreemNisa) [reply content/sre/repository/03-permissions.md]: added

File: content/sre/repository/08-grant-nexus-admin-keycloak.md (hunk 3 above)
Comment (TehreemNisa) [reply content/sre/repository/08-grant-nexus-admin-keycloak.md]: added

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-266-dd5f6706`

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.203`

**Pull request merged**